### PR TITLE
[Cheat] Allow Elegy of Emptiness anywhere

### PR DESCRIPTION
**Pull request by Eblo**
Extremely straightforward, this cheat just circumvents the scene check when playing the song.

image

### Build Artifacts
  - [2ship-linux.zip](https://nightly.link/HarbourMasters/2ship2harkinian/actions/artifacts/2038227911.zip)
  - [2ship-windows.zip](https://nightly.link/HarbourMasters/2ship2harkinian/actions/artifacts/2038236160.zip)
  - [2ship-mac.zip](https://nightly.link/HarbourMasters/2ship2harkinian/actions/artifacts/2038276916.zip)

### CODE REVIEW

**Review by garrettjoecox — APPROVED** [review]
Only point of feedback would be on the VB name, it's not horrible but still reads odd to me "Should elegy in ikana only?"

Otherwise just needs updating to modern menu and headers LGTM

**Comment by Eblo** [comment]
I applied those changes. Agreed on the VB name, so I changed it to `VB_ELEGY_CHECK_SCENE`.

I'm not sure why generating the o2r failed in the PR. It succeeded in my fork's run. I don't have the authority to clear cache or re-run the PR job here, but I'm assuming it was a fluke.

File: mm/2s2h/BenGui/SearchableMenuItems.h [hunk 1]
```diff
@@ -967,6 +967,12 @@ void AddEnhancements() {
                 WIDGET_CVAR_CHECKBOX,
                 {},
                 [](widgetInfo& info) { RegisterMoonJumpOnL(); } },
+              { "Elegy of Emptiness Anywhere",
+                "gCheats.ElegyAnywhere",
+                "Allows Elegy of Emptiness outside of Ikana",
+                WIDGET_CVAR_CHECKBOX,
+                {},
+                [](widgetInfo& info) { RegisterMoonJumpOnL(); } },
```
Comment (garrettjoecox) [inline mm/2s2h/BenGui/SearchableMenuItems.h]: Might need to re-run prettier after this
```suggestion
                {} },
```

Comment (Eblo) [reply mm/2s2h/BenGui/SearchableMenuItems.h]: Whoops. Removed.

**Pull request merged**